lib/crit_tables/slash_critical_table.rb

```diff
@@ -0,0 +1,2505 @@
+                                  :crippled        => false,
+                                  :sleeping        => false,
+                                  :dazed           => false,
+                                  :limb_favored    => false,
+                                  :roundtime       => 0,
+                                  :silenced        => false,
+                                  :slowed          => false,
+                                  :wound_rank      => 1,
+                                  :secondary_wound => nil,
+                                  :regex           => /Weak slash to .*?'s lower back./ },
+                           2 =>
+                                { :type            => "Slash",
+                                  :location        => "Back",
+                                  :rank            => 2,
+                                  :damage          => 10,
+                                  :position        => nil,
+                                  :fatal           => false,
+                                  :stunned         => 0,
+                                  :amputated       => false,
+                                  :crippled        => false,
+                                  :sleeping        => false,
+                                  :dazed           => false,
+                                  :limb_favored    => false,
+                                  :roundtime       => 0,
+                                  :silenced        => false,
+                                  :slowed          => false,
+                                  :wound_rank      => 1,
+                                  :secondary_wound => nil,
+                                  :regex           => /Feint to the left goes astray as .*? dodges./ },
+                           3 =>
+                                { :type            => "Slash",
+                                  :location        => "Back",
+                                  :rank            => 3,
+                                  :damage          => 15,
+                                  :position        => nil,
+                                  :fatal           => false,
+                                  :stunned         => 2,
+                                  :amputated       => false,
+                                  :crippled        => false,
+                                  :sleeping        => false,
+                                  :dazed           => false,
+                                  :limb_favored    => false,
+                                  :roundtime       => 0,
+                                  :silenced        => false,
+                                  :slowed          => false,
+                                  :wound_rank      => 1,
+                                  :secondary_wound => nil,
+                                  :regex           => /Slash (?:to the|along) .*?'s lower back./ },
```

Comment:
this one is having trouble with the xml since `the` was in the group.

`/Slash (?:to|along) .*?'s lower back./` works with both the xml and nonxml